setup.py

```diff
@@ -268,72 +268,16 @@ def make_release_tree(self, base_dir, files):
             package_data[key] = list(set(val + package_data[key]))
 
 setup(  # Finally, pass this all along to setuptools to do the heavy lifting.
-    name="matplotlib",
-    description="Python plotting package",
     author="John D. Hunter, Michael Droettboom",
     author_email="matplotlib-users@python.org",
```

Comment:
author and author_email can be moved across to `pyproject.toml`, something like

https://github.com/bokeh/bokeh/blob/e3aad12fce42660786965ed0093913ddc0fcf80b/pyproject.toml#L23-L26